### ISSUE TITLE
Windows Outlook: Editing a single event within a Recurrence, Office.context.mailbox.item returns series Master ID

### ISSUE THREAD

**Issue by thoffmann-fms** [OfficeDev/office-js#3117]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

# Provide required information needed to triage your issue
Works correctly in Office Online and MacOS, broken in Windows Outlook.

## Your Environment

* Platform PC
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Microsoft® Outlook® for Microsoft 365 MSO (Version 2301 Build 16.0.16026.20002) 64-bit 
* Operating System: Windows 10
* Browser (if using Office on the web): Any
 
## Expected behavior

When editing a single occurrence of a series, Office.context.mailbox.item itemID should be the ID of the edited event.

## Current behavior

Windows returns the Series MasterID.  OWA and Mac return the occurrence Id.

## Steps to reproduce

1. Create a daily 5-day series
2. Save
3. Open the 3rd occurrence only, run an add-in that requests the Item ID.

## Context

User our Add-In makes changes to the entire series rather than just the occurrence the user requested.

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by CurtisRiva** [comment]
We have come across the same issue in Outlook Windows Desktop.

**Comment by nancy-wang** [comment]
Thanks @CurtisRiva and @thoffmann-fms for reporting this issue! Tagging @exextoc to take a look.

**Comment by thoffmann-fms** [comment]
HI, any update? We have several unhappy customers waiting on this fix.

**Comment by thoffmann-fms** [comment]
Is it fixed in New Outlook?
